js_tests/admin/core.test.js

```diff
@@ -40,10 +40,21 @@ QUnit.test('Date.getTwoDigitSecond', function(assert) {
     assert.equal(new Date(2014, 6, 1, 0, 0, 20).getTwoDigitSecond(), '20', '12:00:20 am is 20');
 });
 
+QUnit.test('Date.getAbbrevMonthName', function(assert) {
+    assert.equal(new Date(2020, 0, 26).getAbbrevMonthName(), 'Jan', 'jan 26');
+    assert.equal(new Date(2020, 9, 26).getAbbrevMonthName(), 'Oct', 'oct 26');
+});
+
+QUnit.test('Date.getFullMonthName', function(assert) {
+    assert.equal(new Date(2020, 0, 26).getFullMonthName(), 'January', 'jan 26');
+    assert.equal(new Date(2020, 9, 26).getFullMonthName(), 'October', 'oct 26');
+});
+
```

Comment:
These tests are not necessary, IMO.

Reply:
@felixxm I asked about adding these tests on [django-core-mentorship mailing list](https://groups.google.com/g/django-core-mentorship/c/r4JSPAiFuLU/m/kRTtrOoGAwAJ). @adamchainz suggested to add tests for individual code paths and I agree with Adam on adding individual tests for each individual method.

Reply:
We can keep them, I moved unrelated tests to a separate commit.